### ISSUE TITLE
Improving the solvers in MiniSolver

### ISSUE THREAD

**Issue by jtostie** [gahansen/Albany#15]
In general, improving default solver parameters.

**Comment by ikalash** [comment]
The incorporation of ROL into MiniSolver may have helped with this.  @lxmota , have you compared ROL vs. inherent MiniSolver solvers for unconstrained problems.

**Comment by lxmota** [comment]
@ikalash @jtostie @jwfoulk @calleman21  I think that Jake is referring to the default values for solver parameters that currently exist in MiniSolver. For instance, the trust region solver has been barely exercised and we really have no idea whether the default parameters are good for our particular set of problems.

I have not done a one-to-one comparison between MiniSolver and ROL for the methods that exist in both. I pulsed Jay and Coleman whether they would be interested in trying MiniSolver's interface to ROL. It appears that at the moment they have other priorities, like improving Schwarz's performance. So I have delayed doing that comparison.
